### ISSUE TITLE
Report not being generated when unit test fails

### ISSUE THREAD

**Issue by ciceroneves** [Hargne/jest-html-reporter#4]
Hi, 
I started using jest-html-reporter and the report is nicely generated when all the tests succeed.
However, I made a small change in one of my tests in order to break it and no report is being generated.

Is there something I may be doing wrong or this can be a bug?
Here is my unit test:

```javascript
describe('CheckInRedux reducers', () => {

  it('should return the initial state', () => {
    expect(
        reducer(undefined, {})
    ).toEqual(
        {
          checkInConfirmationData: {
            appointments: {},
            equipmentsWithLoadSummary: [],
            equipmentsWithLocationCode: [],
            equipmentsWithSealsInfo: []
          },
          isFetching: true,
          checkInSucceeded: false,
          error: false
        }
    )
  });
```

The line I changed was `isFetching: true`, it is supposed to be false.

Thanks in advance.

**[UPDATE]**
I think the issue is related to having `"includeFailureMsg": true` in package.json.
Once I set it to false the report was generated but then I cannot see the error message.
Here is a stack trace outputted when includeFailureMsg is set to true.

```
at Object.&lt;anonymous&gt; (/Volumes/Unix/workplace/SesameMobile/src/SesameMobile/App/Redux/CheckIn/CheckInRedux.spec.js:13:1)
    at Object.&lt;anonymous&gt; (/Volumes/Unix/workplace/SesameMobile/src/SesameMobile/node_modules/jest-jasmine2/build/jasmine-async.js:42:32)
    at attemptAsync (/Volumes/Unix/workplace/SesameMobile/src/SesameMobile/node_modules/jest-jasmine2/vendor/jasmine-2.5.2.js:1984:24)
    at QueueRunner.run (/Volumes/Unix/workplace/SesameMobile/src/SesameMobile/node_modules/jest-jasmine2/vendor/jasmine-2.5.2.js:1939:9)
    at /Volumes/Unix/workplace/SesameMobile/src/SesameMobile/node_modules/jest-jasmine2/vendor/jasmine-2.5.2.js:1966:16
    at /Volumes/Unix/workplace/SesameMobile/src/SesameMobile/node_modules/jest-jasmine2/vendor/jasmine-2.5.2.js:1909:9
    at Object.fn (/Volumes/Unix/workplace/SesameMobile/src/SesameMobile/node_modules/jest-jasmine2/build/jasmine-async.js:68:11)
    at attemptAsync (/Volumes/Unix/workplace/SesameMobile/src/SesameMobile/node_modules/jest-jasmine2/vendor/jasmine-2.5.2.js:1984:24)
    at QueueRunner.run (/Volumes/Unix/workplace/SesameMobile/src/SesameMobile/node_modules/jest-jasmine2/vendor/jasmine-2.5.2.js:1939:9)
    at QueueRunner.execute (/Volumes/Unix/workplace/SesameMobile/src/SesameMobile/node_modules/jest-jasmine2/vendor/jasmine-2.5.2.js:1927:10) at index 7
    at XMLStringifier.module.exports.XMLStringifier.assertLegalChar (/Volumes/Unix/workplace/SesameMobile/src/SesameMobile/node_modules/xmlbuilder/lib/XMLStringifier.js:153:15)
    at XMLStringifier.assertLegalChar (/Volumes/Unix/workplace/SesameMobile/src/SesameMobile/node_modules/xmlbuilder/lib/XMLStringifier.js:4:59)
    at XMLStringifier.module.exports.XMLStringifier.eleText (/Volumes/Unix/workplace/SesameMobile/src/SesameMobile/node_modules/xmlbuilder/lib/XMLStringifier.js:33:19)
    at new XMLText (/Volumes/Unix/workplace/SesameMobile/src/SesameMobile/node_modules/xmlbuilder/lib/XMLText.js:17:35)
    at XMLElement.module.exports.XMLNode.text (/Volumes/Unix/workplace/SesameMobile/src/SesameMobile/node_modules/xmlbuilder/lib/XMLNode.js:165:15)
    at XMLElement.module.exports.XMLNode.node (/Volumes/Unix/workplace/SesameMobile/src/SesameMobile/node_modules/xmlbuilder/lib/XMLNode.js:157:15)
    at XMLElement.module.exports.XMLNode.element (/Volumes/Unix/workplace/SesameMobile/src/SesameMobile/node_modules/xmlbuilder/lib/XMLNode.js:102:28)
    at XMLElement.module.exports.XMLNode.ele (/Volumes/Unix/workplace/SesameMobile/src/SesameMobile/node_modules/xmlbuilder/lib/XMLNode.js:357:19)
    at test.failureMessages.forEach (/Volumes/Unix/workplace/SesameMobile/src/SesameMobile/node_modules/jest-html-reporter/index.js:74:20)
    at Array.forEach (native)
```

**Comment by Hargne** [comment]
Hi!
Thanks for your message :)
Which version of jest-html-reporter are you using? I did a bug fix to `includeFailureMsg` pretty recently. If you are on the latest (0.1.2) then there's something not working as intended and I'll have a look at it!

**Comment by ciceroneves** [comment]
Hey thanks for replying :)

I am using version 0.1.0. When I last checked this was the latest version. I will upgrade it and let you know!

Cheers,
Cicero

**Comment by ciceroneves** [comment]
Issue has been fixed in version 1.0.2
You can close this.
